--- origsrc/unison-2.40.61/Makefile	2010-10-08 10:44:59.000000000 -0500
+++ src/unison-2.40.61/Makefile	2010-10-15 16:17:31.006373500 -0500
@@ -60,7 +60,7 @@ STATIC=false
 # NAME, VERSION, and MAJORVERSION, automatically generated
 -include Makefile.ProjectInfo
 
-Makefile.ProjectInfo: mkProjectInfo.ml $(wildcard ../.bzr/branch/last-revision)
+Makefile.ProjectInfo: mkProjectInfo $(wildcard ../.bzr/branch/last-revision)
 	./mkProjectInfo > $@
 
 mkProjectInfo: mkProjectInfo.ml
